vincentlanglet/twig-cs-fixer/0.6/post-install.txt

```diff
@@ -0,0 +1,4 @@
+<bg=green;fg=white></>
+<bg=green;fg=white> You can edit the `.twig-cs-fixer.php` to your needs see</>
+<bg=green;fg=white> https://github.com/VincentLanglet/Twig-CS-Fixer/blob/main/docs/configuration.md </>
```

Comment:
```suggestion
<bg=green;fg=white> See https://github.com/VincentLanglet/Twig-CS-Fixer/blob/main/docs/configuration.md </>
```

Reply:
Thank you for the feedback i implemented it :+1: